docker/shibboleth/sp-cert.pem

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Again, should this be protected? Also it might be worth exposing the cert via an environment variable and writing it to a file in the entrypoint. If the shibboleth CAs are ever rotated, we'll have to rebuild the image.

Reply:
This cert is also public on the `/Shibboleth.sso/Metadata` route of any Shibboleth SP. However, the key for it is provided through an environment variable.

Reply:
Would it be worth splitting `platform-run` into two parts? Not sure if it makes sense to separate it into entrypoint and cmd.

Reply:
Sorry; didn't realize you were doing CMD not ENTRYPOINT. I mean that this should be put as part of `platform-run`. I don't like burning certs into images in general. Really hurts their reusability for other purposes.

Reply:
This image is specific to Penn Labs' platform server and I doubt it would be much use to anyone else, but I agree. I'll remove the hard-coded cert.